src/commands/kick/kick.ts

```diff
@@ -0,0 +1,61 @@
+import { Command, SafetyJim } from '../../safetyjim/safetyjim';
+import * as Discord from 'discord.js';
+
+class Kick implements Command {
+    public usage = 'kick @user [reason] - Kicks the user with the specified reason';
+
+    // tslint:disable-next-line:no-empty
+    constructor(bot: SafetyJim) {}
+
+    public run(bot: SafetyJim, msg: Discord.Message, args: string): boolean {
+        this.runAsync(bot, msg, args);
+        return;
+    }
+
+    private async runAsync(bot: SafetyJim, msg: Discord.Message, args: string): Promise<void> {
+        args = args.split(' ').slice(1).join(' ');
+
+        if (msg.mentions.users.size === 0) {
+            msg.channel.send('You need to mention the user to kick.');
+            return;
+        }
+
+        let member = msg.guild.member(msg.mentions.users.first());
+
+        if (!member || !member.kickable || msg.member.highestRole.comparePositionTo(member.highestRole) <= 0) {
+            msg.channel.send('The specified member is not kickable.');
+            return;
+        }
+
+        // tslint:disable-next-line:max-line-length
+        bot.log.info(`${member.user.tag} was kicked by ${msg.author.tag} (${args || 'No reason specified'})`);
+        member.kick(args || 'No reason specified'); // Audit log compatibility :) (Known Caveat: sometimes reason won't appear, or add if reason has symbols.)
+        bot.database.createUserKick(member.user, msg.author, msg.guild, args || 'No reason specified');
+
+        let db = await bot.database.getGuildConfiguration(msg.guild);
+
+        // tslint:disable-next-line:max-line-length
+        if (!db || !db.ModLogChannelID || !db.ModLogActive || bot.client.channels.get(db.ModLogChannelID).type !== 'text') {

```

Comment:
There will always be ModLogChannelID so no need to check here, you can just check for active